cmd/spiffe-aws-assume-role/cli/cli.go

```diff
@@ -29,7 +36,7 @@ type CliContext struct {
 func (c *CredentialsCmd) Run(context *CliContext) error {
 	spiffeID, err := spiffeid.FromString(c.SpiffeID)
 	if err != nil {
-		return err
+		return errors.Wrap(err, fmt.Sprintf("failed to parse SPIFFE ID from %s", c.SpiffeID))
```

Comment:
The "Wrap" part is misleading, what I'm really doing is capturing the stack trace so that it's available for debugging later. When I was initially adding tests to the package the native error values would contain messages but not the stack traces, which made debugging difficult. errors.Wrap fixes that and makes debugging super easy.